spikeextractors/sortingextractor.py

```diff
@@ -102,7 +112,7 @@ def set_unit_spike_features(self, unit_id, feature_name, value):
                 if unit_id not in self._unit_features.keys():
                     self._unit_features[unit_id] = {}
                 if isinstance(feature_name, str) and len(value) == len(self.get_unit_spike_train(unit_id)):
-                    self._unit_features[unit_id][feature_name] = np.asarray(value)
+                    self._unit_features[unit_id][feature_name] = value  # np.asarray(value) this is to keep memmap object
```

Comment:
@colehurwitz any specific reason why this was cast as array? is it for lists?
I removed the cast to enable memmap arrays, but I can add a check for lists

Reply:
I don't think there was much reason for casting it. Since spiketrains are numpy arrays, I think I wanted to keep the 1-1 correspondence between features and spike trains. I can't remember any other reason why.

Reply:
ok then I'll keep this way. As long as they have the same length, it's ok